simpleclient/pom.xml

```diff
@@ -54,6 +54,24 @@
       <version>4.13.2</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.mockito</groupId>
+          <artifactId>mockito-inline</artifactId>
+          <version>3.12.4</version>
+          <scope>test</scope>
+      </dependency>
```

Comment:
Not needed?

Reply:
Alas, it seems to be needed - https://github.com/webcompere/system-stubs#dependencies .

Reply:
I removed the dependency, and the test works just fine. I don't understand why it is needed.

Anyway, if the `system-stubs` dependency is hard to understand, it might be good not to use it. A few of the options how to make the code testable do not include modifying environment variables at runtime:
* Use reflection to set the flag
* Write an integration test with Testcontainers.

Reply:
Yes, that is the expected output for JVM 11. 

Quoting from the link above:

> ⚠ WARNING: JDK Compatibility. From JDK16 onwards, there are deeper restrictons on the ability to use reflection. Previous versions of this library, and others in the space, encounter an Illegal Reflective Access warning, or even a runtime error such as java.lang.reflect.InaccessibleObjectException when trying to manipulate the Map behind the system's environment variables.

Consequently, this library now uses mockito-inline version 3.x to enable the interception of calls for reading environment variables. This requires consumers to both use a compatible version of Mockito AND be prepared for the inline implementation of Mockito mocks.

Note: Groovy users may need Mockito >= 4.5.0 for compatibility.